prog/dftb+/lib_extlibs/mbd.F90

```diff
@@ -0,0 +1,23 @@
+#:include "common.fypp"
+
+!> Imports the functionality of libMBD.
+module dftbp_mbd
+#:if WITH_MBD
+  use mbd, TMbdInit => mbd_input_t, TMbd => mbd_calc_t
+#:endif
+  implicit none
+  private
+
+  public:: TMbdInit, TMbd
+
+#:if not WITH_MBD
+  ! Dummy empty type
+  type :: TMbdInit
+  end type TMbdInit
+
+  ! Dummy empty type
+  type :: TMbd
+  end type TMbd
+#:endif
```

Comment:
Some compilers can be picky about empty derived types, especially together with `pointer` and `associated` checks, not sure how this affects their behaviour for `allocatable` and `allocated` checks.